suite-native/app/src/ModalsRenderer.tsx

```diff
@@ -0,0 +1,21 @@
+import { AlertRenderer } from '@suite-native/alerts';
+import { PassphraseModalRenderer } from '@suite-native/passphrase';
+import { ToastRenderer } from '@suite-native/toasts';
+import { BiometricsModalRenderer } from '@suite-native/biometrics';
+import { NotificationRenderer } from '@suite-native/notifications';
+
+import { Snow } from './snow/Snow';
+
+export const ModalsRenderer = () => (
+    <>
+        <AlertRenderer />
+        <PassphraseModalRenderer />
+        <ToastRenderer />
+        <Snow />
+        <BiometricsModalRenderer />
+        {/* Notifications are disabled until the problem with after-import notifications flooding is solved. */}
+        {/* More here: https://github.com/trezor/trezor-suite/issues/7721  */}
+        {/* <NotificationRenderer> */}
+        <NotificationRenderer />
```

Comment:
Is it on purpose that NotificationRenderer is no more commented out?

Reply:
🫢🫢🫢
Definitely not correct.

Will fix.

Reply:
https://github.com/trezor/trezor-suite/pull/10801/commits/4db4ecf300cce3fe63e97ee4aa2cb6b32cc05e34